algorithms/ar-isrt/c/insertion_sort.c

```diff
@@ -0,0 +1,32 @@
+# include <stdio.h>
+
+// Insertion sort function
+void insertionSort(int arr[], int len) {
+	int i, key;
```

Comment:
Codacy found an issue: [The scope of the variable 'key' can be reduced.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6277928)